src/app/modules/main/communities/tokens/models/token_item.nim

```diff
@@ -14,19 +14,22 @@ type
     chainName*: string
     chainIcon*: string
     accountName*: string
+    remainingSupply*: int
```

Comment:
Is this same thing than `remainingTokens` in `token_model`? Why these names are different?

Reply:
Fixed